src/Apps/Order/Routes/__tests__/Offer.jest.enzyme.tsx

```diff
@@ -178,6 +178,7 @@ describe("Offer InitialMutation", () => {
       const page = new OrderAppTestPage(wrapper)
 
       expect(page.transactionSummary.text()).toContain("Your offer")
+      await page.selectRandomPriceOption() // hack to initialize the page's offervalue, otherwise it resets to undefined on selection below
```

Comment:
this test was a bit misleading since the selection of the first option only worked because there was a default (i.e: selecting the second option at the beginning of the test would fail).

Now that we don't have the default, any first selection would end up in `undefined` being the value initialised, can't put my finger on why that is the case. Not happy but also not sure it's worth spending more time on it since we're developing a new checkout.